qemu/tests/cfg/seabios_iommu.cfg

```diff
@@ -0,0 +1,42 @@
+# Please add "iommu=pt intel_iommu=on" to host kernel line
+- seabios_iommu: install setup image_copy unattended_install.cdrom
+    virt_test_type = qemu
+    type = seabios_iommu
+    only q35
+    no WinXP WinVista Win7 Win8 Win8.1 Win2003
+    no Win2008 Win2008..r2 Win2012 Win2012..r2
+    extra_params = "-device intel-iommu,device-iotlb=on,intremap=on"
+    machine_type_extra_params = "kernel-irqchip=split"
+    virtio_dev_iommu_platform = on
+    virtio_dev_ats = on
+    images += " stg0 stg1"
+    image_name_stg0 = "images/stg0"
+    image_name_stg1 = "images/stg1"
+    image_size_stg0 = 1G
+    image_size_stg1 = 1G
+    data_image_size = 1G
```

Comment:
What's the usage of this param? Why not directly use `image_size`?

Reply:
Updated, thanks!